cetmix_tower_server/models/cx_tower_reference_mixin.py

```diff
@@ -15,6 +15,15 @@ class CxTowerReferenceMixin(models.AbstractModel):
     _name = "cx.tower.reference.mixin"
     _description = "Cetmix Tower reference mixin"
 
+    # List of models that should try to generate references based on the related
+    # model reference.
+    # Eg flight plan lines references are generated based on the flight plan one.
+    # {model_name: [parent_model, relation_field, custom_suffix]}
+    GENERATE_FROM_RELATED = {
+        "cx.tower.plan.line": ["cx.tower.plan", "plan_id", "line"],
+        "cx.tower.plan.line.action": ["cx.tower.plan.line", "line_id", "action"],
+        "cx.tower.variable.value": ["cx.tower.variable", "variable_id", "variable"],
+    }
```

Comment:
this model is a mixin which can be applied to any model, I recommend not to use hardcode model binding in mixin. it complicates code maintenance, and you need to change the source mixin every time you inherit this mixin in new models. As a simple solution, you should instead make a new method that will return this dictionary, and in each model that inherits this mixin extend this method.